Makefile

```diff
@@ -33,15 +33,15 @@ metalinter_test: ensure_tools
 		--enable=goconst \
 		--enable=gosimple \
 	 	--enable=ineffassign \
-	   	--enable=interfacer \
+	  --enable=interfacer \
```

Comment:
why did all this spacing get weird?

Reply:
Mh, some had two spaces instead of tabs. Everything is now intended with 2 tabs.